src/main/resources/application-cloud.yml

```diff
@@ -60,6 +60,10 @@ spring:
       min-idle: 3
     continue-on-error: true
     schema: classpath:/schema-cloud.sql
+    user: ${cloud.services.postgresql.connection.username}
+    password: ${cloud.services.postgresql.connection.password}
+    default-schema: sample
```

Comment:
This should be collection exercise.

Reply:
Testing this branch with `default-schema: collectionexercise` on prunes.